src/docs/page-components.jsx

```diff
@@ -4,36 +4,24 @@ import Avatars from '../components/avatar/pages/avatar-interactive';
 import Bubbles from '../components/bubble/pages/bubble-interactive';
 import Badges from '../components/badges/pages/badges-interactive';
 import Buttons from '../components/buttons/pages/buttons-interactive';
+import IconsAsButtons from '../components/icon-as-button/pages/icon-as-button-interactive';
+import slugify from 'slugify';
```

Comment:
I'm not sure, but you have bad library.
You should use local file src/docs/slugify.js